docs/getting-started.md

```diff
@@ -372,9 +376,9 @@ If you want to sync using a public testnet such as Kovan, Rinkeby, or Ropsten, j
 ### 2.4 Deploying The Subgraph
 
 When you deploy the subgraph to the Graph Node, it will start ingesting all the subgraph events from the blockchain, transforming that data with the subgraph mappings, and storing it in the Graph Node. 
-Note that running a subgraph can safely be stopped and started, and it will pick up where it has left off. 
+Note that a running subgraph can safely be stopped and started; it will pick up where it has left off.
 
-Now you can run `yarn deploy --watch` in the subgraph directory. `yarn deploy` should have been added to `package.json` in section 3.3. This can also be found in [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This will compile the mappings, and deploy the mappings, schema and the subgraph manifest to IPFS. The subgraph is now fully running!
+Now that the infrastructure is setup you can run `yarn deploy --watch` in the subgraph directory. `yarn deploy` should have been added to `package.json` in section 1.3 when we took a moment to go through the setup for [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This will compile the mappings, and deploy the mappings, schema and subgraph manifest to IPFS. The subgraph is now fully running!
```

Comment:
`set up`

Reply:
There's one step missing here. I'd make it `This will build the subgraph, upload its output files to IPFS and deploy it to the Graph Node.`